nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import errno
+import argparse
+
+def parse_args(args=None):
+    Description = 'Reformat {{ cookiecutter.name }} samplesheet file and check its contents.'
+    Epilog = """Example usage: python check_samplesheet.py <FILE_IN> <FILE_OUT>"""
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument('FILE_IN', help="Input samplesheet file.")
+    parser.add_argument('FILE_OUT', help="Output file.")
+    return parser.parse_args(args)
+
+
+def make_dir(path):
+    if not len(path) == 0:
+        try:
+            os.makedirs(path)
+        except OSError as exception:
+            if exception.errno != errno.EEXIST:
+                raise
```

Comment:
You want to print the exception, so you need to pass the `exception` variable. Otherwise you'll just get an uninformative "generic raise" message.

```suggestion
                raise exception
```